frontend/src/pages/detail/components/PlatformSection/PlatformSection.tsx

```diff
@@ -0,0 +1,18 @@
+import IconBadgeList from "@shared/components/IconBadgeList/IconBadgeList";
+import SectionTitle from "../SectionTitle";
+import * as S from "./PlatformSection.styled";
+
+interface PlatformSectionProps {
+  platforms: string[];
+}
```

Comment:
```suggestion
interface PlatformSectionProps {
  platforms: Platform[];
}
```

저희 상수화를 다 맞춰놔서 `string[]` 보다는 플랫폼들에 대한 타입을 맞춰놔야 나중에 백엔드 api 받아와서 props으로 넣을 때도 타입 안전성이 더 좋을 것 같습니다~